src/extension/module.js

```diff
@@ -2865,6 +2901,9 @@
       // toggle sidekick
       window.hlx.sidekick.toggle();
     }
+    if(window.hlx.sidekick.isEditor()){
+      sampleRUM('sidekick:open', true);
```

Comment:
```suggestion
      sampleRUM('sidekick:open', {
             source: '', // TODO: the current URL being edited or previewed (this can be the word or google docs URL)
          });
```